apps/backend/prisma/migrations/20240904214932_/migration.sql

```diff
@@ -1,5 +1,5 @@
 -- CreateEnum
-CREATE TYPE "Role" AS ENUM ('BODY_ADMIN', 'BODY_MEMBER', 'USER');
+CREATE TYPE "Role" AS ENUM ('SUPERUSER', 'BODY_ADMIN', 'BODY_MEMBER', 'USER');
```

Comment:
prisma will want to wipe the db if you migrate with `yarn prisma migrate dev`, but that's fine locally.
On junior, we should migrate with `yarn prisma migrate deploy`, it will thrown a warning, but everything should be fine since this change was applied to the prod db